ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/ConsistentPackageContentView.kt

```diff
@@ -31,15 +31,20 @@ internal fun ConsistentPackageContentView(
 internal fun ConsistentPackageContentView(
     packages: List<TemplateConfiguration.PackageInfo>,
     selected: TemplateConfiguration.PackageInfo,
+    shouldAnimate: Boolean = true,
```

Comment:
I thought about changing the implementation to just have a single text view instead of multiple as we have now, but that seemed a more complex implementation... So I decided to just disable the animation.